### PR TITLE
DAOS-5535 control: nvme health display tidy

### PR DESCRIPTION
**Pull request by tanabarr**
Improve presentation of NVMe stats for a given SSD in the control plane
by fixing the display of unix timestamps and only printing bio rw stats
when collecting over drpc (not when fetching through go bindings).

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

### CODE REVIEW

File: src/control/cmd/dmg/pretty/storage_test.go [hunk 1]
```diff
@@ -97,12 +106,7 @@ PCI:%s Model:%s FW:%s Socket:%d Capacity:%s
     Power Cycles:%d
     Power On Duration:%s
     Unsafe Shutdowns:%d
-    Error Count:%d
```
Comment (sydneyvanda) [inline src/control/cmd/dmg/pretty/storage_test.go:100]: this error count is specific to NVMe device stats right? ie not apart of BIO in-memory error counters I don't think. we can probably keep this in the display if so
Actually I am confused what this Error Count actually is?

Comment (tanabarr) [reply src/control/cmd/dmg/pretty/storage_test.go:100]: it's not used/populated anywhere . that's why I removed it, seems to be a surplus parameter which at some point was probably confused with  error log count

**Review by mjmac — APPROVED** [review]

**Comment by mjmac** [comment]
It seems fine to me... What is the rationale for using UTC in the timestamps, though?

**Comment by mjmac** [comment]
@sydneyvanda: Are you OK with this PR landing?

**Review by kjacque — APPROVED** [review]
LGTM

**Comment by tanabarr** [comment]
> It seems fine to me... What is the rationale for using UTC in the timestamps, though?

None, should be printing in local time zone, will correct in https://github.com/daos-stack/daos/pull/3589 , set it for testing but didn't switch back. thanks

**Comment by tanabarr** [comment]
known failures in CI:

DAOS-5784
DAOS-5749
DAOS-5745

**Review by NiuYawei — APPROVED** [review]

**Comment by sydneyvanda** [comment]
> @sydneyvanda: Are you OK with this PR landing?

@mjmac sorry for the delay. Thanks for landing!

**Pull request merged**